worlds/animal_well/options.py

```diff
@@ -180,6 +186,7 @@ class AnimalWellOptions(PerGameCommonOptions):
     wheel_tricks: WheelTricks
     weird_tricks: WeirdTricks
     exclude_song_chests: ExcludeSongChests
+    deathlink: Deathlink
```

Comment:
```suggestion
    death_link: DeathLink
```
I think all the other games do it as death_link rather than deathlink